### DISCUSSION TITLE
Pan programatically on button click in React

### DISCUSSION THREAD

**Discussion by stefann01** [chartjs/chartjs-plugin-zoom#872] [Q&A]
Seems like `chart.pan({x: amount})` doesn't work, where chart is a React ref to a bar chart

**Comment by kurkle** [comment]
Could you provide more details please (a reproduction would be best to get someone to look at this). I would guess React is re-rendering the chart and losing the pan in the process.

**Comment by ghost** [comment]
I have the same issue in pure html project!
